fatal-errors.json

```diff
@@ -634,6 +637,18 @@
       "categoryId": "configuration",
       "kbLinkId": "ITwinSpatialAlignment",
       "canUserFix": true
+    },
+    "QuartzJobDataMapNotCreatedCorrectly": {
+      "description": "Value from quartz job data map is null for some keys.",

```

Comment:
As a user-facing string, I don't think they will or should know about "quartz"... can we use something more generic that better suggests they need to contact us? Something like Internal job scheduling error maybe...

Reply:
I confirmed within team and for now we have decided to remove this error `QuartzJobDataMapNotCreatedCorrectly `because we don't throw it at the moment.  We will separately include it in the documentation in future if required.